src/mockServiceWorker.js

```diff
@@ -38,14 +38,14 @@ self.addEventListener('message', async function (event) {
 
   switch (event.data) {
     case 'KEEPALIVE_REQUEST': {
-      sendToClient(client, {
+      await sendToClient(client, {
```

Comment:
Do you find any particular benefit of awaiting this? The "message" event is not tied to the request handling pipeline in any way, so what we introduce here are all side effects. We may await them if we depend on async actions execution order but in this case we don't.